Tools/autotest/pysim/util.py

```diff
@@ -173,6 +177,9 @@ def pexpect_close(p):
     global close_list
 
     ex = None
+    if p is None:
```

Comment:
Who is calling this with`p` as None?

Reply:
On mac with display, the gdb lldb is launched using `subprocess.Popen` instead of pexpect.